src/org/skypro/skyshop/exceptions/BestResultNotFound.java

```diff
@@ -0,0 +1,18 @@
+package org.skypro.skyshop.exceptions;
+
+import java.io.IOException;
+
+public class BestResultNotFound extends IOException {
```

Comment:
IOException == Input Output Exception. Это базовое исключение для ошибок ввода/вывода. В нашем случае нет ввода и вывода, нужно наследоваться от Exception
```suggestion
public class BestResultNotFound extends Exception {
```